src/ConnectAuth.ts

```diff
@@ -121,9 +174,33 @@ export async function verifyRequest<E>({
 
   // Load existing installation
   const unverifiedConnectJwt = decodeUnverifiedConnectJwt(rawConnectJwt);
+
   const credentials = await credentialsLoader(unverifiedConnectJwt.iss);
   if (!credentials) {
-    throw new AuthError('Unknown issuer', { code: AuthErrorCode.UNKNOWN_ISSUER });
+    throw new AuthError('Unknown issuer', {
+      code: AuthErrorCode.UNKNOWN_ISSUER,
+      unverifiedConnectJwt,
+    });
+  }
+
+  // Check for a signed uninstallation
```

Comment:
`verifyRequest` also needs the option to check for signed installs because the uninstallation lifecycle event is also signed asymmetrically.

I thought about adding a third method `verifyUninstallation` to prevent overloading with `verifyRequest` but at the end that would duplicate a lot of code.

Thoughts?